ports/mimxrt/boards/MIMXRT1062_af.csv

```diff
@@ -123,3 +123,6 @@ GPIO_SD_B1_08,USDHC2_DATA4,FLEXSPI_A_DATA0,LPUART7_TXD,SAI1_TX_BCLK,LPSPI2_SOUT,
 GPIO_SD_B1_09,USDHC2_DATA5,FLEXSPI_A_DATA1,LPUART7_RXD,SAI1_TX_SYNC,LPSPI2_SIN,GPIO3_IO09,,,,,,,ALT5
 GPIO_SD_B1_10,USDHC2_DATA6,FLEXSPI_A_DATA2,LPUART2_RXD,LPI2C2_SDA,LPSPI2_PCS2,GPIO3_IO10,,,,,,,ALT5
 GPIO_SD_B1_11,USDHC2_DATA7,FLEXSPI_A_DATA3,LPUART2_TXD,LPI2C2_SCL,LPSPI2_PCS3,GPIO3_IO11,,,,,,,ALT5
+WAKEUP,,,,,,GPIO5_IO00,,NMI,,,,,ALT5
+PMIC_ON_REQ,SNVS_PMIC_ON_REQ,,,,,GPIO5_IO01,,,,,,,ALT0
+PMIC_STBY_REQ,CCM_PMIC_STBY_REQ,,,,,GPIO5_IO02,,,,,,,ALT0
```

Comment:
Are these real GPIO pins, should they start with `GPIO_`?

I see it's called `WAKEUP_DIG` in one of the other boards.

Reply:
> Are these real GPIO pins, should they start with `GPIO_`?

Yes those are connected to physical pins, and they support GPIO function, but no they don't start with `GPIO`, the names used for all pins in this file is the pad name from the datasheet (the other ones just happen to start with `GPIO`.

> I see it's called `WAKEUP_DIG` in one of the other boards.

`WAKEUP` is the correct pad name used in the RT1060/2 reference manuals.

Reply:
For 117x there's `_DIG` suffix, which has been taken care of.